pom.xml

```diff
@@ -302,8 +302,8 @@
         <spring42.version>4.2.9.RELEASE_1</spring42.version>
         <spring43.version>4.3.25.RELEASE_1</spring43.version>
         <spring50.version>5.0.15.RELEASE_1</spring50.version>
```

Comment:
Ideally these shouldn’t these be upgraded too (5.0.16 addressed the same CVE, and 4.3.27 includes [a backport of related fixes](https://github.com/spring-projects/spring-framework/issues/24580)), but there are no corresponding ServiceMix bundles.

Reply:
Yeah, I saw that there were no corresponding bundles, or I would have included those too.

Reply:
I'm preparing SMX bundles releases about that.